htdocs/salaries/paiement_salary.php

```diff
@@ -274,7 +274,7 @@
 				print img_picto("Auto fill", 'rightarrow', "class='AutoFillAmount' data-rowid='".$namef."' data-value='".($objp->amount - $sumpaid)."'");
 			}
 			$remaintopay = $objp->amount - $sumpaid;
-			print '<input type=hidden class="sum_remain" name="'.$nameRemain.'" value="'.$remaintopay.'">';
+			print '<input type=hidden class="sum_remain" name="'.$nameRemain.'" value="'.GETPOST('amount_'.$objp->id, 'intcomma').'">';
```

Comment:
The first time we entry the page, value will not be prefilled.
May be you should make here :
GETPOSTISSET($namef) ? GETPOST($namef, 'alpha') : remaintopay